package.json

```diff
@@ -48,11 +48,12 @@
     "@angular/router": "^3.1.0",
     "@types/jasmine": "^2.5.35",
     "@types/node": "^6.0.45",
+    "angular2-template-loader": "^0.5.0",
     "autoprefixer": "^6.5.1",
     "awesome-typescript-loader": "^2.2.4",
     "babel-core": "^6.17.0",
-    "babel-polyfill": "^6.16.0",
     "babel-loader": "^6.2.5",
+    "babel-polyfill": "^6.16.0",
```

Comment:
We are using it in a couple of places. First one is to compile the angular es6 distribution. The other is as the second step of our typescript pipeline to enable language features not available when the typescript compiler targets es5.